src/test/java/com/eppo/sdk/EppoClientTest.java

```diff
@@ -40,654 +39,246 @@
 public class EppoClientTest {
 
   private static final int TEST_PORT = 4001;
-
-  private WireMockServer mockServer;
-  private static final ObjectMapper MAPPER = new ObjectMapper();
-
-  static {
-    MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-  }
-
-  @Data
-  static class SubjectWithAttributes {
-    @JsonProperty String subjectKey;
-    @JsonProperty EppoAttributes subjectAttributes;
-  }
-
-  @Data
-  static class AssignmentTestCase {
-    @JsonProperty String experiment;
-
-    @JsonDeserialize(using = AssignmentValueTypeDeserializer.class)
-    AssignmentValueType valueType = AssignmentValueType.STRING;
-
-    @JsonProperty List<SubjectWithAttributes> subjectsWithAttributes;
-    @JsonProperty List<String> subjects;
-    @JsonProperty List<String> expectedAssignments;
-  }
-
-  enum AssignmentValueType {
-    STRING("string"),
-    BOOLEAN("boolean"),
-    JSON("json"),
-    NUMERIC("numeric");
-
-    private final String strValue;
-
-    AssignmentValueType(String value) {
-      this.strValue = value;
-    }
-
-    String value() {
-      return this.strValue;
-    }
-
-    static AssignmentValueType getByString(String str) {
-      for (AssignmentValueType valueType : AssignmentValueType.values()) {
-        if (valueType.value().compareTo(str) == 0) {
-          return valueType;
-        }
-      }
-      return null;
-    }
+  private static final String TEST_HOST = "http://localhost:" + TEST_PORT;
+  private static WireMockServer mockServer;
+
+  private static final String DUMMY_FLAG_API_KEY = "dummy-flags-api-key"; // Will load flags-v1
+  private static final String DUMMY_BANDIT_API_KEY =
+      "dummy-bandits-api-key"; // Will load bandit-flags-v1
+  private AssignmentLogger mockAssignmentLogger;
+  private BanditLogger mockBanditLogger;
+
+  @BeforeAll
+  public static void initMockServer() {
+    mockServer = new WireMockServer(TEST_PORT);
```

Comment:
Unlike the shared common repository, these test cases use a mock local server, which makes them much faster. 🐎